encoding/da.go

```diff
@@ -623,3 +608,39 @@ func CheckBatchCompressedDataCompatibility(batch *Batch, codecVersion CodecVersi
 	}
 	return codec.CheckBatchCompressedDataCompatibility(batch)
 }
+
+// GetChunkEnableCompression returns whether to enable compression for the given block height and timestamp.
+func GetChunkEnableCompression(codecVersion CodecVersion, chunk *Chunk) (bool, error) {
```

Comment:
Just for my understanding: this looks similar to the removed `GetMaxChunksPerBatch`. But here we need this logic because we need to check these things on `Chunk` (not `DAChunk`) right?